languages/python/default.nix

```
@@ -49,6 +49,79 @@ in {
       defaultText = lib.literalExpression "pkgs: [ ]";
       example = lib.literalExpression "pkgs: [ pkgs.requests ]";
     };
+    config = lib.mkOption {
+      type = with lib.types; nullOr lines;
+      description = ''
+        Python interpreter startup configuration. See
+        <https://docs.python.org/3/using/cmdline.html#envvar-PYTHONSTARTUP>
+        for details.
+      '';
+      default = null;
+      defaultText = lib.literalExpression "null";
+      example = lib.literalExpression ''
+        import numpy as np
+      '';
+    };
+    configPath = lib.mkOption {
+      type = lib.types.path;
+      description = ''
+        Python interpreter startup configuration file path. See
+        <https://docs.python.org/3/using/cmdline.html#envvar-PYTHONSTARTUP>
+        for details.
+      '';
+      default = "${config.xdg.configHome}/python/startup.py";
+      defaultText = "$XDG_CONFIG_HOME/python/startup.py";
+      example =
+        lib.literalExpression
+        ''"${config.xdg.configHome}/pythonrc.py"'';
+    };
+    historyPath = lib.mkOption {
+      type = with lib.types; nullOr path;
+      description = ''
+        Python interpreter history file path. See
+        <https://docs.python.org/3.13/using/cmdline.html#envvar-PYTHON_HISTORY>
+        for details.
+        This option is only available in Python 3.13.
+      '';
+      default = null;
+      defaultText = lib.literalExpression "null";
+      example = lib.literalExpression ''
+        "${config.xdg.stateHome}/python_history"
+      '';
+    };
+    enableColors =
+      lib.mkEnableOption ''
+        colors in the interpreter. See
+        <https://docs.python.org/3.13/using/cmdline.html#envvar-PYTHON_COLORS>
+        for details.
+        This option is only available in Python 3.13.
+      ''
+      // {default = true;};
+  };
+  config.home = {
+    packages = lib.mkIf cfg.enable [cfg.package];
+    sessionVariables = lib.mkMerge [
+      (lib.mapAttrs
+        (_: toString)
+        (lib.filterAttrs (_: var: var != [] && var != null) {
+          PYTHONSTARTUP = cfg.configPath;
+          PYTHON_HISTORY = cfg.historyPath;
+        }))
+      (lib.mkIf (!cfg.enableColors) {PYTHON_COLORS = "-1";})
+    ];
+    file."${cfg.configPath}" = lib.mkIf (cfg.config != null) {
+      text = ''
+        """Python interpreter configuration file.
+
+        Generated by Home Manager, do not edit manually.
+        See: https://docs.python.org/3/using/cmdline.html#envvar-PYTHONSTARTUP
+
+        This script is executed at the start of each interactive python console
+        when it is defined in the `PYTHONSTARTUP` environment variable.
+        """
```

I suggest the following wording:
```suggestion
        # DO NOT EDIT -- this file has been generated automatically.
        # Python interpreter startup commands, generated via home-manager.
```

In particular I would avoid using a multiline string as a comment here as it gets picked up by Python as a docstring and gets stored as the `__doc__` variable.